cmd/root.go

```diff
@@ -129,6 +129,9 @@ func init() {
 	rootCmd.Flags().StringVar(&testRepo, "test-repo", "", "skip specific tests. allows regular expressions.")
 	viper.BindPFlag("test-repo", rootCmd.Flags().Lookup("test-repo"))
 
+	rootCmd.Flags().StringSlice("extra-conformance-test-args", []string{}, "Additional parameters to be provided to the conformance container. These parameters should be specified as key-value pairs, separated by commas. Each parameter should start with -- (e.g., --list-labels=true,--allowed-not-ready-nodes=2)")
```

Comment:
I would prefer the flag name to just be `extra-args`. Your explanation here is great and should be clear to users that these args are passed to to the conformance pod.

Reply:
fixed.